packages/context/src/context-view.ts

```diff
@@ -28,23 +29,36 @@ const nextTick = promisify(process.nextTick);
  * points. For example, the RestServer can react to `controller` bindings even
  * they are added/removed/updated after the application starts.
  *
+ * `ContextView` is an event emitter that emits the following events:
+ * - 'open': when the view is open (starting to observe context events)
```

Comment:
How do you envision to use this event in practice? I believe `view.open` is called by the framework before the view is injected to the receiver and thus before the receiver can install `on('open')` handler.

Reply:
Good catch. I'll remove `open` and `close` events as I only have use cases for `refresh/resolve` in mind.

Reply:
I removed `open` event.